core/templates/pages/release-coordinator-page/release-coordinator-page.component.ts

```diff
@@ -46,16 +64,34 @@ export class ReleaseCoordinatorPageComponent implements OnInit {
   promoBarConfigForm!: FormGroup;
   memoryCacheDataFetched: boolean = false;
   submitButtonDisabled: boolean = false;
+  directiveSubscriptions = new Subscription();
+  loadingMessage: string = '';
+  newUserGroupName: string = '';
+  separatorKeysCodes: number[] = [ENTER];
+  userGroupsBackup!: Map<string, UserGroup>;
```

Comment:
Could you add a comment explaining what this variable stores and describing the lifecycle for backups?

Reply:
Updated